controller/WebController.php

```diff
@@ -337,23 +337,47 @@ public function invokeGlobalSearch($request)
         $vocabObjects = array();
         if ($vocids) {
             foreach($vocids as $vocid) {
-                $vocabObjects[] = $this->model->getVocabulary($vocid);
+                try {
+                    $vocabObjects[] = $this->model->getVocabulary($vocid);
+                } catch (Exception $e) {
```

Comment:
An empty catch block is bad practice. What happens with the `$vocabObjects` variable that is used below?
At a minimum, it could be set to `null` here, or `array()` if that makes more sense.